release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3392,6 +3392,41 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.14.44
+[id="ocp-4-14-44_{context}"]
+=== RHSA-2025:0029 - {product-title} 4.14.44 bug fix and security update
+
+Issued: 09 January 2024
+
+{product-title} release 4.14.44, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2025:0029[RHSA-2025:0029] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2025:0032[RHBA-2025:0032] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.14.44 --pullspecs
+----
+
+[id="ocp-4-14-44-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, the Single-Root I/O Virtualization (SR-IOV) Operator did not expire the acquired lease during the Operator's shutdown operation. This impacted a new instance of the Operator, because the new instance had to wait for the lease to expire before the new instance was operational. With this release, an update to the Operator shutdown logic ensures that the Operator expires the lease when the Operator is shutting down. (link:https://issues.redhat.com/browse/OCPBUGS-44726[*OCPBUGS-44726*])
+
+* Previously, when you attempted to use {olm-first} to upgrade an Operator, the upgrade was blocked and an `error validating existing CRs against new CRD's schema` message was generated. An issue existed with {olm}, whereby {olm} erroneously identified incompatibility issues validating existing custom resources (CRs) against the new Operator version's custom resource definitions (CRDs). With this release, the validation is corrected so that Operator upgrades are no longer blocked. (link:https://issues.redhat.com/browse/OCPBUGS-46595[*OCPBUGS-46595*]) 
+
+* Previously, the `aws-sdk-go-v2` software development kit (SDK) failed to authenticate an `AssumeRoleWithWebIdentity` API operation on an {aws-first} {sts-first} cluster. With this release, `pod-identity-webhook` now includes a default region, and this issue no longer persists. (link:https://issues.redhat.com/browse/OCPBUGS-46487[*OCPBUGS-46487*])
```

Comment:
1. Suggestion to double-check if `now` is needed here, considering that the sentence already starts with `With this release`: https://redhat-documentation.github.io/supplementary-style-guide/#now.
2. Just wondering if `persists` is the best word choice here (could using `persists` be understood in the sense that this issue still appears but doesn't linger?).
```suggestion
* Previously, the `aws-sdk-go-v2` software development kit (SDK) failed to authenticate an `AssumeRoleWithWebIdentity` API operation on an {aws-first} {sts-first} cluster. With this release, `pod-identity-webhook` includes a default region, which prevents such authentication failures. (link:https://issues.redhat.com/browse/OCPBUGS-46487[*OCPBUGS-46487*])
```